src/DiffEditor/__snapshots__/index.spec.tsx.snap

```diff
@@ -0,0 +1,181 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
might wanna double check these snapshots in some diff-checker but i think they're correct from some manual checking

Reply:
https://www.diffchecker.com/fdb1wij4/